src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -2691,19 +2691,19 @@ public static IntPtr GetFunctionPointerForDelegate<TDelegate>(TDelegate d)
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         internal static extern IntPtr GetFunctionPointerForDelegateInternal(Delegate d);
 
-#if FEATURE_LEGACYSURFACE
-
-#if FEATURE_COMINTEROP
         [System.Security.SecurityCritical]  // auto-generated_required
         public static IntPtr SecureStringToBSTR(SecureString s) {
             if( s == null) {
                 throw new ArgumentNullException(nameof(s));
             }
             Contract.EndContractBlock();
             
-            return s.ToBSTR();
-        }
+#if FEATURE_COMINTEROP
+            return s.MarshalToBSTR();
+#else
+            return IntPtr.Zero;
```

Comment:
Should this throw PNSE instead of return zero? It would be nice to ifdef the whole method, and move the stub that throws PNSE to NonPortable.cs.